workflow-templates/im-run-validate-deployed-terraform.yml

```diff
@@ -67,6 +66,7 @@ jobs:
         working-directory: '${{ env.TF_WORKING_DIR }}'
 
     steps:
+      # TODO:  SWAT - Need a replacement for determining which release is the latest
```

Comment:
Just a heads up, I left a TODO here because I think it's kind of an involved fix.  I looked at the way IFP did it and it seems like this could be simplified.  I'd like to work on a more streamlined solution before updating this file.  Maybe part of that is investigating whether this template is providing the value we want.